libraries/bezier-rs/src/subpath/transform.rs

```diff
@@ -325,6 +326,19 @@ impl<ManipulatorGroupId: crate::Identifier> Subpath<ManipulatorGroupId> {
 		Some((clipped_subpath1, clipped_subpath2.unwrap()))
 	}
 
+	/// Returns a subpath that results from rotating the path around the origin by the given angle (in radians).
+	/// <iframe frameBorder="0" width="100%" height="375px" src="https://graphite.rs/bezier-rs-demos#subpath/rotate/solo" title="Rotate Demo"></iframe>
+	pub fn rotate(&self, angle: f64) -> Subpath<ManipulatorGroupId> {
```

Comment:
I'm wondering if it'd be cleaner to use the `apply_transform` function that 0HyperCube added to do this transformation instead.